controllers/storagecluster/storageclasses.go

```diff
@@ -367,6 +374,10 @@ func newNonResilientCephBlockPoolStorageClassConfiguration(initData *ocsv1.Stora
 		},
 		isClusterExternal: initData.Spec.ExternalStorage.Enable,
 	}
+	if disableKeyRotation {
```

Comment:
The same piece of code is in 2 places. I think we can refactor it into a single method and reuse it. it.

Reply:
Done